.github/workflows/release.yml

```diff
@@ -20,19 +20,20 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Setup Bintray
+      - name: Setup Maven Central
         uses: actions/setup-java@v2
         with:
           java-version: '8'
           distribution: 'adopt'
-          server-id: bintray
-          server-username: BINTRAY_USER
-          server-password: BINTRAY_PASS
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+          gpg-private-key: ${{ secrets.GPG_KEY_CONTENTS }}
```

Comment:
This is a bit of a guess but I assume we want the actual key here and not the key id or something else.

Reply:
That variable is actually the file but base64 encoded. So it's the password encoded file that contains the key. See also the document I wrote in which I tried to explain that.